docs/_articles/en/user-guide/perf-enhancements.md

```diff
@@ -3,37 +3,32 @@ title: Performance Enhancements
 lang: en
 ---
 
-The Apex, Deploy and Retrieve commands are in the process of moving to a new architecture. These commands have been rearchitected into independent Typescript libraries and CLI plugins as part of our larger effort to break up the salesforce-alm plugin.  [You can read more about that in this blog post](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021.html). There’s many great benefits to this effort and one of the best is better performance. For single file deploys and retrieves, you'll likely notice a significant performance gain when using the latest architecture.
+The Apex, Deploy and Retrieve commands are in the process of moving to a new architecture. These commands have been rearchitected into independent Typescript libraries and CLI plugins as part of our larger effort to break up the salesforce-alm plugin. [You can read more about that in this blog post](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021.html). There’s many great benefits to this effort and one of the best is better performance. For single file deploys and retrieves, you'll likely notice a significant performance gain when using the latest architecture.
 
 ## Execution Process and Status
 
-This architecture change is done through a phased approach.  For both the Apex library and Deploy/Retrieve library, we are rolling out the changes through these phases:
+This architecture change is done through a phased approach. For both the Apex library and Deploy/Retrieve library, we are rolling out the changes through these phases:
```

Comment:
```suggestion
This architecture change is done through a phased approach. For the Deploy/Retrieve library, we are rolling out the changes through these phases:
```

Reply:
Well, maybe we can keep it as is ?

Reply:
I think we can leave it as it is for now- I sync'd with Stephanie and she recommended only removing the reference from the 'Setup' section but leaving the rest as it is in case customers have questions